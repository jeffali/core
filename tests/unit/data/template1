This is

a template


[%CFEngine linux:: %]
on Linux


[%CFEngine BEGIN %]
with newlines.
[%CFEngine END %]

something at the end.
