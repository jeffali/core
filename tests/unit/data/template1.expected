This is

a template


on Linux


with newlines.

something at the end.
